fastlane/Fastfile

```diff
@@ -20,8 +20,14 @@ before_all do
 
   # Check that the env files exist
   unless is_ci || File.file?(USER_ENV_FILE_PATH)
-    UI.user_error!("~/.simplenoteios-env.default not found: Please copy env/user.env-example to #{USER_ENV_FILE_PATH} and fill in the values")
+    error_message = <<~ERROR
+    ~/.simplenoteios-env.default not found. Please copy env/user.env-example to #{USER_ENV_FILE_PATH} and fill in the values") using the following command:
+
+    \tcp #{PROJECT_ROOT_FOLDER}/fastlane/env/user.env-example #{USER_ENV_FILE_PATH}
+    ERROR
+    UI.user_error!(error_message)
```

Comment:
This looks like this when printed:

<img width="1723" alt="image" src="https://user-images.githubusercontent.com/1218433/200631799-0bd600f2-1cf2-4799-8c4f-54f087cc201b.png">